cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
+	"name": "Using fixtures to represent data",
+	"email": "hello@cypress.io",
+	"body": "Fixtures are a great way to mock data for responses to routes"
+}
```

Comment:
We probably don’t want to add this file?

Reply:
ugh, I should probably add this to `.gitignore` Cypress is creating it automatically

Reply:
Done